src/schema/objects/metadata.yaml

```diff
@@ -70,6 +70,20 @@ AnalyticalApproach:
     - type: array
       items:
         type: string
+AnatomicalImage:
+  name: AnatomicalImage
+  display_name: Anatomical Image
+  description: |
+    The path(s) to the anatomical MR image file(s), if present,
+    to which the associated MRS data file corresponds.
+    Contains one or more [BIDS URIs](SPEC_ROOT/common-principles.md#bids-uri).
+    Using forward-slash separated paths relative to the participant subdirectory is
+    [DEPRECATED](SPEC_ROOT/common-principles.md#definitions).
```

Comment:
Is there a reason to write in deprecated functionality to a new extension?

Reply:
Removed.